src/modules/powerrename/dll/dllmain.cpp

```diff
@@ -316,8 +315,6 @@ class PowerRenameModule : public PowertoyModuleIface
 
     void save_settings()
     {
-        CSettingsInstance().SetEnabled(m_enabled);
-        CSettingsInstance().Save();
         Trace::EnablePowerRename(m_enabled);

```

Comment:
No, we just use the same general settings file to store the information whether the module is enabled or not.

Reply:
So, m_enabled will till have the right information regarding the enabled state?

Reply:
Yes, the DSC module must close PowerToys when applying the settings to avoid interfering with PowerToys state. Then we'll get the actual state in `init_settings()`

Reply:
But if someone disables in Settings will this get changed as well? Just wanting to still send the right telemetry.

Reply:
Yes, because `disable` will be called that changes this value.